index.html

```diff
@@ -58,6 +58,161 @@ <h3>Let's connect</h3>
+            <li><span>html</span></li>
+            <li><span>css</span></li>
+            <li><span>javascript</span></li>
+          </ul>
+          <button type="button" class="button button-enabled">See Project</button>
+        </div>
+
+        <div class="flex works-grid-item">
+          <img class="width-100" src="images/works/printing.png" alt="Printing press website screenshot">
+          <h3 class="section-title margin-top-12">Printing made easy</h3>
+          <div class="work-period">
+            <ul>
+              <li class="text-color-primary">Canopy</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p class="work-description text-color-primary margin-top-20">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="work-categories margin-top-12 padding-0">
+            <li><span>html</span></li>
+            <li><span>css</span></li>
+            <li><span>javascript</span></li>
+          </ul>
+          <button type="button" class="button button-enabled">See Project</button>
+        </div>
+      </div>
+    </section>
+
+    <section class="about-myself padding-top-114">
+      <div class="flex about-wrap">
+        <h2 class="section-title">About myself</h2>
+        <p class="headline-profile text-color-primary margin-top-12">I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href="#" class="headline-cta margin-top-12">
+          <h3>Let's connect</h3>
+        </a>
+        <ul class="social-icons margin-top-12">
+          <li>
+            <a href="#">
+              <img class="social twitter" src="images/icons/twitter-icon.png" alt="Twitter icon">
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img class="social linkedin" src="images/icons/linkedin-icon.png" alt="Twitter icon">
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img class="social m" src="images/icons/m-icon.png" alt="Twitter icon">
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img class="social github" src="images/icons/github-icon.png" alt="Twitter icon">
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img class="social f" src="images/icons/f-icon.png" alt="Twitter icon">
+            </a>
+          </li>
+        </ul>
+        <button type="button" class="button button-enabled">See my resume</button>
+
+        <div class="other-info">
+          <ul class="flex other-info-wrap padding-0">
+            <li class="other-info-list">
+              <div class="flex li-flex wrap">
+                <h4>Language</h4>
+                <span> > </span>
```

Comment:
- [ ] Kindly add the right icon to this header tag,so your design look exactly like the figma design  👍🏿 



Figma Design          |  Your design    
:-------------------------:|:-------------------------:
![Screenshot_35](https://user-images.githubusercontent.com/69638013/164500798-5190cf1a-c05b-4568-af0f-69be1abc34fa.png) | ![Screenshot_33](https://user-images.githubusercontent.com/69638013/164500515-34a1cbd0-17b7-40ff-9e9b-4c25b7be5243.png)